my-app/src/Components/task/task.tsx

```diff
@@ -27,44 +27,44 @@ const Task = (props: IProps) => {
       setIsEditTitle(true);
     };
     const handleEditDescr = () => {
-        setIsEditDescr(true);
-      };
+      setIsEditDescr(true);
+    };
   
     const handleCorrectTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
       setCorrectedTitle(e.target.value);
     };
 
     const handleCorrectDescr = (e: React.ChangeEvent<HTMLInputElement>) => {
-        setCorrectedDescr(e.target.value);
-      };
+      setCorrectedDescr(e.target.value);
+    };
   
     const handleSaveTitle = () => {
       setIsEditTitle(false);
       dispatch(editTaskTitle(taskId, correctedTitle));
     };
 
     const handleSaveDescr = () => {
-        setIsEditDescr(false);
-        dispatch(editTaskDescr(taskId, correctedDescr));
-      };
+      setIsEditDescr(false);
```

Comment:
Looks like there is a problem with formatting, it works differently for some of you. That's why we have these spaces here, please check